ee/clickhouse/queries/funnels/funnel_correlation.py

```diff
@@ -122,17 +113,24 @@ def __init__(
             # Funnel Step by default set to 1, to give us all people who entered the funnel
 
         # Used for generating the funnel persons cte
-        self._funnel_actors_generator = ClickhouseFunnelActors(
-            Filter(
-                data={
-                    key: value
-                    for key, value in self._filter.to_dict().items()
-                    # NOTE: we want to filter anything about correlation, as the
-                    # funnel persons endpoint does not understand or need these
-                    # params.
-                    if not key.startswith("funnel_correlation_")
-                }
-            ),
+
+        filter_data = {
+            key: value
+            for key, value in self._filter.to_dict().items()
+            # NOTE: we want to filter anything about correlation, as the
+            # funnel persons endpoint does not understand or need these
+            # params.
+            if not key.startswith("funnel_correlation_")
+        }
+        # NOTE: we always use the final matching event for the recording because this
+        # is the the right event for both drop off and successful funnels
+        filter_data.update(
+            {"include_final_matching_events": self._filter.include_recordings,}
+        )
+        filter = Filter(data=filter_data)
+        funnel_order_actor_class = get_funnel_order_actor_class(filter)
```

Comment:
Small bug fix around ensuring the correlation person modal is respecting the `funnel_order`

Reply:
good catch